src/frontend/src/modules/SolarHomeSystem/OwnerWidget.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <widget :title="$tc('words.owner')" color="green" :subscriber="'shs-owner'">
+    <md-list class="md-double-line">
+      <md-list-item :to="`/people/${person.id}`">
+        <div class="md-list-item-text">
+          <span>{{ $tc("words.name") }}</span>
+          <span>{{ person.name }} {{ person.surname }}</span>
+        </div>
+        <md-icon>arrow_forward</md-icon>
+      </md-list-item>
+    </md-list>
+  </widget>
+</template>
+
+<script>
+import Widget from "@/shared/widget"
+
+export default {
+  name: "OwnerWidget",
```

Comment:
In other components we don't really use Widget in the name. So, let's follow that convention and remove `...Widget` from here.